src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -78,7 +56,7 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
 
     $ids = [];
 
-    if ($this->configuration['handler_settings']['field_mode'] == 'admin') {
+    if ($this->isGroupAdmin()) {
```

Comment:
Now confused about how this relates to changes in https://github.com/amitaibu/og/pull/11/files#diff-669ced81d1fae6b8b9a940fcf8dc8d35R81 :) Can we remove the field_mode stuff?